shell/edit/cis.cattle.io.clusterscanbenchmark.vue

```diff
@@ -30,7 +34,9 @@ export default {
   },
 
   async fetch() {
-    this.configMaps = await this.$store.dispatch('cluster/findAll', { type: CONFIG_MAP });
+    if (!paginationUtils.isEnabled({ rootGetters: this.$store.getters }, { store: 'cluster', resource: { id: CONFIG_MAP } })) {
```

Comment:
store action
- if pagination enabled, return none else all

this.configMaps = await this.$store.dispatch('cluster/conditionalFindAllWithPagintion')  (tbd name)